client/src/pages/Restaurant/FoodList/FoodCard.js

```diff
@@ -1,5 +1,57 @@
-function FoodCard() {
-  return <div>FoodCard</div>;
+function FoodCard({ name, price, description, imageUrl }) {
+  const classes = {
+    cardContainer: {
+      boxSizing: "border-box",
+      width: "100%",
+      height: "68px",
+      border: "2px solid #5a5a5a",
+      padding: "2px",
+      display: "flex",
+      gap: "0.25rem",
+      backgroundColor: "hsl(35,0%,97%)",
+    },
+    foodInfo: {
+      maxWidth: "740px",
+    },
+    foodName: {
+      fontSize: "1rem",
+      fontWeight: "bold",
+      overflow: "hidden",
+      whitespace: "nowrap",
+      textOverflow: "ellipsis",
+      height: "30%",
+    },
+    foodPrice: {
+      fontSize: "1rem",
+      marginLeft: "auto",
+      padding: "15px",
+    },
+    foodDescription: {
+      fontSize: "0.8rem",
+      overflow: "auto",
+      whiteSpace: "normal",
+      textOverflow: "ellipsis",
+      height: "45px",
+    },
+    image: {
+      backgroundColor: "#F0EAD6",
+      width: "42px",
+      objectFit: "contain",
```

Comment:
Why do we need a background color?
Looks like it's missing height 100%

Reply:
When an image doesn't fit its container (not all images are sqaure), I put a light easy on the eyes background color to sort of "fill-in" the rest of the div and it provides some consistant sizing of the image area I feel.